### PR TITLE
1 hour SLA updates

### PR DESCRIPTION
**Pull request by virginiaulrich**

### CODE REVIEW

File: handbook/company/customer-first.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ This is how this all fits together in terms of workflow:
...
+Currently, CEs are responsible for answering all how-to questions and handling all feature requests (whether these come up pre- or post-sales). Similarly, CS is responsible for troubleshooting something not working correctly (whether these come up pre- or post-sales), taking into account the prioritization considerations as provided by CEs, and adhering to our contractual [SLAs](https://about.sourcegraph.com/handbook/support#slas) post-sales and 1 hour internal SLA response time pre-sales between 2am-5pm PT Monday-Friday (this will expand to 24/5 support as the team grows over the course of FY22). As CS continues to hire during FYQ2 and new folks are onboarded, CS will also take on responsibility for how-to questions and feature requests for non-strategic customers.
```
Comment (abeatrix) [inline handbook/company/customer-first.md:36]: > As CS continues to hire during FYQ2 

Is FYQ2 correct or FY22?

**Review by abeatrix — COMMENTED** [review]
have a question but other than that, LGTM!

File: handbook/company/customer-first.md (hunk 1 above)
Comment (virginiaulrich) [reply handbook/company/customer-first.md:36]: It is actually FYQ2 ... Aimee and are still working out the details, but the hiring we did is in support of that.

**Review by abeatrix — APPROVED** [review]

File: handbook/company/customer-first.md (hunk 1 above)
Comment (abeatrix) [reply handbook/company/customer-first.md:36]: just making sure :D approved! Thanks V!

**Review by airamare01 — COMMENTED** [review]
Should we also include UTC time on L16

**Review by airamare01 — APPROVED** [review]
If I understand correctly, 
In Step 6, we will need to submit as open first, and then, proceed to Step 7, submit as open again and lastly change the requester.

May I ask if this order is to be strictly adhered to? Asking because previous triaging order might have been a little different for me but with intended results

Other than that, this is crystal clear!

**Comment by virginiaulrich** [comment]
> Should we also include UTC time on L16

We can do both! We need to list pacific since that is how we write our SLAs in our contracts. Having both doesn't hurt. I'll update that.

**Comment by virginiaulrich** [comment]
> If I understand correctly,
> In Step 6, we will need to submit as open first, and then, proceed to Step 7, submit as open again and lastly change the requester.
> 
> May I ask if this order is to be strictly adhered to? Asking because previous triaging order might have been a little different for me but with intended results
> 
> Other than that, this is crystal clear!

I don't think that is necessary, actually. I changed it. What do you think of the revisions?

**Review by airamare01 — APPROVED** [review]
Thank you for passing the changes V! This looks good!

**Review by jasonhawkharris — APPROVED** [review]

**Review by donavonelli — APPROVED** [review]

**Review by benjaminwgordon — APPROVED** [review]

**Review by Kelvin-Lee — APPROVED** [review]
Read through it and nothing jumped out at me as problematic.

**Review by rhiam — APPROVED** [review]

**Review by topebali — APPROVED** [review]
I think this is very clear and responsibilities are well outlined. I have not objections, I approve.

**Review by DaedalusG — APPROVED** [review]
LGTM!

**Comment by amberfurbush0317** [comment]
Looks good. no question I approve!

**Review by amberfurbush0317 — APPROVED** [review]
Looks good i approve

**Review by gabtorre — COMMENTED** [review]
Everything makes sense and is clear

**Review by gabtorre — APPROVED** [review]

**Review by hicksca — APPROVED** [review]

File: handbook/company/customer-first.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ This is the area of responsibility for each organization:
 
 * Sales is responsible for establishing, retaining, and growing our commercial relationship with customers.
 * Customer Engineering (aka CE) is responsible for educating our users on how to use Sourcegraph, providing proactive technical expertise, and is overall accountable for our customers’ success, usage, and adoption of our products.
-* Effective 2021-08-08, Customer Support (aka CS) is responsible for satisfying support SLAs for customers pre-sales (1 hour) and post-sales (defined contractual [SLAs](https://about.sourcegraph.com/handbook/support#slas)) between 2am-5pm Pacific Time Monday-Friday (this will expand to 24/5 support as the team grows over the course of FY22). Between now and 2021-08-08, as the CS team hires and onboards additional folks, it will adhere to a 24 hour SLA for all pre-sales issues between 9am-5pm PT Monday-Friday.
+* Customer Support (aka CS) is responsible for satisfying support SLAs for customers pre-sales (1 hour) and post-sales (defined contractual [SLAs](https://about.sourcegraph.com/handbook/support#slas)) between 2am-5pm PT (9:00-0:00 UTC) Monday-Friday (this will expand to 24/5 support as the team grows over the course of FY22).
```
Comment (alexAtSourcegraph) [inline handbook/company/customer-first.md:16]: How did the 2am - 5pm PT (9:00-0:00 UTC) time commitment come about? Also if I was a customer, I think the 2am-5pm PT being first and the (9:00-0:00 UTC)  in parentheses shows a bias towards the west coast as opposed to being more general. 

I suggest switching the placement of both or even removing the PT. Unless there is a certain reason for it that I am not privy to.

File: handbook/company/customer-first.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ This is how this all fits together in terms of workflow:
...
+Currently, CEs are responsible for answering all how-to questions and handling all feature requests (whether these come up pre- or post-sales). Similarly, CS is responsible for troubleshooting something not working correctly (whether these come up pre- or post-sales), taking into account the prioritization considerations as provided by CEs, and adhering to our contractual [SLAs](https://about.sourcegraph.com/handbook/support#slas) post-sales and 1 hour internal SLA response time pre-sales between 2am-5pm PT (9:00-0:00 UTC) Monday-Friday (this will expand to 24/5 support as the team grows over the course of FY22). CS will also take on responsibility for how-to questions and feature requests for non-strategic customers in the future.
```
Comment (alexAtSourcegraph) [inline handbook/company/customer-first.md:36]: Same as my previous comment about representing time.

**Review by alexAtSourcegraph — COMMENTED** [review]
I just have a couple of comments about time representation. Other than that it looks great!

File: handbook/company/customer-first.md (hunk 2 above)
Comment (virginiaulrich) [reply handbook/company/customer-first.md:16]: Great question! The reason is that our SLA terms in our contracts are stated in pacific. And this expands upon our contractual SLA.

Comment (virginiaulrich) [reply handbook/company/customer-first.md:16]: But I think it's fair to just say UTC and not overthink it (on my part)

Comment (alexAtSourcegraph) [reply handbook/company/customer-first.md]: Thanks for the insight about the SLA terms being stated in Pacific time!

**Pull request merged**